### PR TITLE
docs: add image directive to public roadmap

### PR DESCRIPTION
**Pull request by AndrewKushnir**
## PR Type
What kind of change does this PR introduce?

- [x] Documentation content changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by mgechev — APPROVED** [review]

File: aio/content/guide/roadmap.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ In Q4 2021 we designed a solution for introducing strict typings for forms and i
 
 In the process of making Angular simpler, we're working on introducing APIs that allow developers to bootstrap applications, instantiate components, and use the router without NgModules. We'll share an RFC to start a design discussion with the community, followed by a series of pull request with the implementation. In the first part of this project we finalized the design of standalone components, directives, and pipes ([RFC](https://github.com/angular/angular/discussions/43784)).
 
+### Improve image performance
+
+The [Aurora](https://web.dev/introducing-aurora/) and the Angular teams started collaboration to design and implement a new directive that would help Angular users to leverage various image optimization techniques. We are in the process of initial research and design.
```
Comment (kara) [inline aio/content/guide/roadmap.md:21]:
```suggestion
The [Aurora](https://web.dev/introducing-aurora/) and the Angular teams have started a collaboration to design and implement a new directive that would help Angular users to leverage various image optimization techniques. We are in the process of initial research and design.
```

**Review by kara — APPROVED** [review]
LGTM!

**Comment by dylhunn** [comment]
@AndrewKushnir This has a merge conflict, mind rebasing?

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit bf98c646f7f718830df0636cef9ecfd0fa606c28.

**Pull request closed without merging**